src/main/resources/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketBuildTrigger/config.jelly

```diff
@@ -45,6 +45,9 @@
   <f:entry title="Cancel outdated jobs?" field="cancelOutdatedJobs">
     <f:checkbox default="false"/>
   </f:entry>
+  <f:entry title="Build chronologically?" field="buildChronologically">
+    <f:checkbox default="false"/>
```

Comment:
And here, the documentation lies and says that the default for the value is false.

Please change the default behavior to NOT include this change, and make sure the documentation matches the actual code.